### ISSUE TITLE
A11y_SQL Azure Data Studio_Add an Azure account_Add an Azure account_Screenreader:Screenreader does not announce the text content provided above the add account button.

### ISSUE THREAD

**Issue by SwatiTiwari-TCS** [microsoft/azuredatastudio#15837]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

GitHubTags: #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.3.1;#NVDA;#JAWS;#STP

### Environment Details:
Application Name: SQL Azure Data Studio 
Version: 1.30.0-insider (user setup)
Windows Version: Win10
Screen reader : JAWS, NVDA2020.2

### Repro Steps:

1. Launch the application-Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over to the 'Azure' header and select the '+' icon.
4. Hover over to the Azure header in the blade that opens up and click on the "+" icon.
5. Open a browser and enter the url as shown in the Website field and then the user code provided in the User Code field. You can also click on the Copy & Open button and select "Open" on the dialog when promped for opening the microsoft.com/devicelogin url, and paste the code (already in the clipboard).
6. Start screen reader and navigate using up/down arrow key to the above the add button and observe its announce the text content or not.

### Actual:
Screen reader does not announce the text content provided above the add account button.

**This issue is observed with both NVDA and JAWS.**

### Expected:
Screen reader should announce the text content provided above the add account button.

### User Impact:
Screenreader only user will miss some information that provided in text form.

**Comment by alanrenmsft** [comment]
as shown in the screenshot below, nvda is announcing it correctly
![image](https://user-images.githubusercontent.com/13777222/129246909-0b356846-d11e-4bf7-a12b-b24c362f70b0.png)

please reopen and attach a video if this is not what you were referring to.

**Comment by Harika1702** [comment]
![image](https://user-images.githubusercontent.com/100270074/217470408-b8dbc591-22ef-4a8e-9231-6f9f8447a96d.png)
Verified the issue in Azure data studio in the below version.
Version: 1.40.2 (user setup)
Commit: 661384637db384fe5d4e6224069adbe708580b16
Date: 2022-12-22T22:17:02.658Z
VS Code: 1.67.0
Electron: 17.4.11
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.14-electron.0
OS: Windows_NT x64 10.0.22623
GitHub Tags: #Closed;
